rerun_py/rerun_sdk/rerun/log/image.py

```diff
@@ -90,6 +101,22 @@ def log_image(
     if interpretable_as_image and num_non_empty_dims != len(shape):
         image = np.squeeze(image)
 
+    if jpeg_quality is not None:
+        try:
+            pil_image = Image.fromarray(image)
```

Comment:
I think this ends up baking in some different assumptions on the allowed dtypes see:

> In the case of NumPy, be aware that Pillow modes do not always correspond to NumPy dtypes. Pillow modes only offer 1-bit pixels, 8-bit pixels, 32-bit signed integer pixels, and 32-bit floating point pixels.

from [here](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.fromarray). I'm also not sure how they interpret linear vs gamma and range depending on the dtype

Reply:
Good catch - I've added an `astype` cast. I will also assume that unless otherwise stated, everyone works in gamma space. It is the norm.